### ISSUE TITLE
Unable to create new subnet without first creating a DNS zone.

### ISSUE THREAD

**Issue by 065673** [cohoe/starrs#262]
Hey Grant, is it possible to create a new subnet without having to create a DNS zone? 
We use infoblox DNS and does not have a TSIG key configured. 
Gracias 
Mike Chelotti

**Comment by cohoe** [comment]
No. But you can create a dummy key, then set DDNS to False for the zone.

**Comment by 065673** [comment]
Ok that worked. 
seem to be having another issue where after I create a new subnet I can't seem to view it. 
Not Found
The requested URL /ip/subnet/view/10.11.12.0/24 was not found on this server.

Apache/2.2.15 (Scientific Linux) Server at 10.2.120.140 Port 80 
Hope all is well with you , hit me up sometime. mchelotti@arbor.net

**Comment by cohoe** [comment]
The netmask / should be %2F in your URL. (a la /ip/subnet/view/10.0.49.0%2F24). If that link came from somewhere it's probably a bug in cohoe/starrs-web, not here. If that was manual, well, user error.

**Comment by 065673** [comment]
Yeah it's in the URL http://10.2.120.140/ip/subnet/view/10.11.12.0%2F24 , I'll enable debugging and see if it's shows more.
